picasso/src/test/java/com/squareup/picasso3/BitmapHunterTest.kt

```diff
@@ -165,14 +165,11 @@ class BitmapHunterTest {
     assertThat(eventRecorder.decodedBitmap).isNull()
   }
 
-  @Test fun huntUnrecognizedUri() {
+  @Test(expected = IllegalStateException::class)
```

Comment:
Please never use this. An earlier statement in the body could throw and your test would still pass.

Reply:
Ah fair enough. Reverting!